### PR TITLE
[FileCommander] improvments 

### PR DESCRIPTION
**Pull request by geeeeeeeeeee**
Fix line selection for move and delete
ReAdd open ViewEdit by startTeletext

### CODE REVIEW

File: lib/python/Plugins/Extensions/FileCommander/plugin.py [hunk 1]
```diff
@@ -305,7 +305,7 @@ def __init__(self, session, pathLeft="", pathRight="", leftActive=None):
 		self["key_blue"] = StaticText()
 		self["key_menu"] = StaticText(_("MENU"))
 		self["key_info"] = StaticText(_("INFO"))
-		self["actions"] = HelpableActionMap(self, ["OkCancelActions", "MenuActions", "InfoActions", "ColorActions", "NavigationActions"], {
+		self["actions"] = HelpableActionMap(self, ["OkCancelActions", "MenuActions", "InfoActions", "ColorActions", "NavigationActions", "InfobarTeletextActions"], {
```
Comment (jbleyel) [inline lib/python/Plugins/Extensions/FileCommander/plugin.py:308]: InfobarTeletextActions is not good

File: lib/python/Plugins/Extensions/FileCommander/plugin.py [hunk 2]
```diff
@@ -325,7 +325,8 @@ def __init__(self, session, pathLeft="", pathRight="", leftActive=None):
 			"right": (self.keyGoRightColumn, _("Switch to the right column")),
 			"down": (self.keyGoLineDown, _("Move down a line")),
 			"pageDown": (self.keyGoPageDown, _("Move down a screen")),
-			"bottom": (self.keyGoBottom, _("Move to last line / screen"))
+			"bottom": (self.keyGoBottom, _("Move to last line / screen")),
+			"startTeletext": (self.keyViewEdit, _("View or edit file (if size < 1MB)"))
```
Comment (jbleyel) [inline lib/python/Plugins/Extensions/FileCommander/plugin.py:329]: There is no protection.
The key is always working and this is not good.

File: lib/python/Plugins/Extensions/FileCommander/plugin.py [hunk 3]
```diff
@@ -652,6 +653,10 @@ def successCallback(job):
 			if "status" in self:
 				self.displayStatus(_("Delete job completed."))
 				self.sourceColumn.refresh()
+				if startIndex >= self.sourceColumn.count():
+					self.sourceColumn.goBottom()
+				else:
+					self.sourceColumn.setCurrentIndex(startIndex)
```
Comment (geeeeeeeeeee) [inline lib/python/Plugins/Extensions/FileCommander/plugin.py:659]: You can extract this in a method too and rename `startIndex`.

File: lib/python/Plugins/Extensions/FileCommander/plugin.py (hunk 2 above)
Comment (geeeeeeeeeee) [reply lib/python/Plugins/Extensions/FileCommander/plugin.py:329]: I want this key always working but

I'm sorry. Feel free to improve. Maybe i can see and learn.

**Pull request merged**